### ISSUE TITLE
Issue: Upgrade OC 8.0.4 to 8.0.5

### ISSUE THREAD

**Issue by apg1980** [owncloud/core#17466]
Hi i am getting a lot of those errors after upgrading to 8.0.5:

Fatal 	core 	Invalid OCS content returned for app 166056 	2015-07-08T06:12:28+02:00
Fatal 	core 	Invalid OCS content returned for app 166048 	2015-07-08T06:12:28+02:00
Fatal	core	Invalid OCS content returned for app 166048	2015-07-08T06:12:28+02:00
Fatal	core	Invalid OCS content returned for app 166049	2015-07-08T06:12:27+02:00
Fatal	core	Invalid OCS content returned for app 166051	2015-07-08T06:12:27+02:00
Fatal	core	Invalid OCS content returned for app 166055	2015-07-08T06:12:26+02:00
Fatal	core	Invalid OCS content returned for app 166046

Those apps are all the onboard apps like pdf_viewer etc.

**Comment by zirydy** [comment]
I get the same error messages since I upgraded to 8.0.5 as well. I think these messages appear when one opens up the "Apps" page (where one can activate/deactivate apps).

**Comment by bit-seq** [comment]
Same log messages for me. Are you two able to activate apps that have to be installed?

Last monday I installed two owncloud instances for testing purposes, where the first was used immediately. App installation went flawlessly. Today I tried to configure the second instance and was not able to install apps, i.e. Contacts, Calendar, Documents. The Activate button changes its text to Deactivate, however, the app won't be listed in the app selection menu and a page reload results in Activate buttons again. Also missing 'Enable only for specific groups' checkboxes are indicating that the installation didn't complete.
